### ISSUE TITLE
Add user preference for clientSideProcessing timeout 

### ISSUE THREAD

**Issue by dskvr** [sandwichfarm/nostr-watch#380]
**Is your feature request related to a problem? Please describe.**
Presently on production, a dynamic timeout is used which determines a good timeout. unfortunately, dependning on network conditions, this can greatly slow down checks. On staging, it's a hardcoded to `15s` (`15000 millis`).  

**Describe the solution you'd like**
Should be user configurable, this way, when they see public relay counts get lower, they have some control. 

**Describe alternatives you've considered**
convert nostr.watch to a text file

**Additional context**
Everyone has their own preferences.
